.github/workflows/bump-agent-versions.sh

```diff
@@ -1,15 +1,16 @@
 #!/bin/bash
```

Comment:
I'm not sure whether we want to mix up the agent versions with the pinned versions.

For instance, see https://github.com/elastic/elastic-agent/pull/8140

It contains versions in the format major.minor.patch.

While the `.stable-snapshot-version` will contains versions in the format `major.minor.patch-hash`. 

I'd expect  `.stable-snapshot-version` to be updated daily while `testing/integration/testdata/.upgrade-test-agent-versions.yml` not that often, maybe a couple of times monthly.

Reply:
I agree with @v1v on this: we should avoid mixing the 2 version bump flows.
The agent versions play a role in our upgrade tests and should change only when a version bump is performed because of a new release.
ECH version could (and in most cases should) change at each unified release run.